.github/workflows/welcome-first-time-contrib.yml

```
@@ -0,0 +1,18 @@
+name: Greeting the new contributor 
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.AUTH_TOKEN }}
```

```suggestion
        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

You where right my apologies. Please revert my previous suggestion.

---

Done!